src/client/actions/MessageReactionAdd.js

```diff
@@ -15,7 +15,7 @@ class MessageReactionAdd extends Action {
   handle(data) {
     if (!data.emoji) return false;
 
-    const user = this.getUser(data);
+    const user = data.member ? this.client.users.add(data.member.user) : this.getUser(data);
```

Comment:
Wouldn't it be more wise to cache the member too? This could be tweaked to use `this.getMember(data.member, guild).user`, although of course, it would require `guild` to be defined and checked beforehand.

Reply:
Since the data is there we could cache it indeed; that would require another change to `this.getMember()` because at the moment `getMember` creates member Objects that only contain a User object, nothing else, as seen in the file change above.